src/ILLink.RoslynAnalyzer/RequiresAnalyzerBase.cs

```diff
@@ -187,6 +239,28 @@ private void ReportRequiresDiagnostic (OperationAnalysisContext operationContext
 				url));
 		}
 
+		private void ReportMismatchInAttributesDiagnostic (SymbolAnalysisContext symbolAnalysisContext, ISymbol member, ISymbol baseMember)
+		{
+			string message = string.Empty;
+			if (!member.HasAttribute (RequiresAttributeName) && baseMember.IsVirtual)
+				message = $"Base member '{baseMember.ToDisplayString ()}' with 'RequiresUnreferencedCodeAttribute' has a derived member '{member.ToDisplayString ()}' without 'RequiresUnreferencedCodeAttribute'";
```

Comment:
These messages should still be in the resources file, they just shouldn't have separate diagnostics associated.